engines/director/picture.cpp

```diff
@@ -0,0 +1,26 @@
+/* ScummVM - Graphic Adventure Engine
+ *
```

Comment:
Do we need this file?

Reply:
I was planning to move the implementations there, to avoid having to include Image::ImageDecoder in this header.